pac4j-oauth/src/main/java/org/pac4j/oauth/client/QQClient.java

```diff
@@ -0,0 +1,106 @@
+package org.pac4j.oauth.client;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.pac4j.oauth.profile.qq.QQProfile;
+import org.pac4j.oauth.profile.qq.QQProfileCreator;
+import org.pac4j.oauth.profile.qq.QQProfileDefinition;
+import org.pac4j.scribe.builder.api.QQApi20;
+
+/**
+ * <p>This class is the OAuth client to authenticate users in Tencent QQ Connect.</p>
+ * <p>It returns a {@link QQProfile}.</p>
+ * <p>More information at http://wiki.connect.qq.com/%E4%BD%BF%E7%94%A8authorization_code%E8%8E%B7%E5%8F%96access_token</p>
+ *
+ * @author Zhang Zhenli
+ * @since 3.1.0
+ */
+public class QQClient extends OAuth20Client<QQProfile> {
+
+    public enum TencentQQScope {
+        /**
+         * Get the nickname, avatar, and gender of the logged in user
+         */
+        get_user_info,
```

Comment:
I'd prefer to have enum in uppercase.

Reply:
In fact, this can be used directly by toString()，

Reply:
OK. Keep them as is then.